### PR TITLE
PR

### PR DESCRIPTION
**Pull request by dnsghd49**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,8 +6,6 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>Hacker Times</title>
-	<link rel="stylesheet" href="style.css"/>
```
Comment (hinchley2018) [inline index.html:9]: We need this now that I fixed the link @dnsghd49

Comment (dnsghd49) [reply index.html:9]: tyty i was in a rush xD

**Review by hinchley2018 — CHANGES_REQUESTED** [review]
the fonts and stuff is good but we need the link in the head

**Review by hinchley2018 — APPROVED** [review]
Great job!

**Pull request merged**